index.js

```diff
@@ -2,6 +2,9 @@ const hamMenu = document.querySelector('.ham-menu');
 const mobileMenu = document.querySelector('.mobile-menu');
 const closeMenu = document.querySelector('.close-icon');
 const mobileLinks = document.querySelectorAll('.mobile-link');
+const email = document.querySelector('.email');
+const form = document.querySelector('.form-1');
+const errorContainer = document.createElement('div');
```

Comment:
- [ ] Please I recommend adding functionality to the `index.js` file in order to empty fields with messages when a field is empty. I recommend adding the error message to the contact form field for other users to actually see the error messages. 😄

Reply:
@braincee 
Please can you elaborate 🙏 this a little more. I am not able to understand what I need to do.